src/04-Fundamentals-2.mdx

```diff
@@ -1,4 +1,121 @@
+
+### Step 5: Start the framework
+
+Now that all the necessary packages are installed, you can run the command `yarn start`. This script command, which is defined in the `package.json` file, runs a gulp task that serves the website locally, so you can always see what you're coding. Now that the framework is served, you can start coding components. Good luck! 
+
+#### Resources
+
++ [Node.js Guide for Frontend Developers](https://blog.bloomca.me/2018/06/21/nodejs-guide-for-frontend-developers.html)
++ [Introduction to gulp](https://developers.google.com/web/ilt/pwa/introduction-to-gulp)
+
+#### TODO: Adjust section for 'bugs' in framework.
+
+## SCSS
+
+Now that everything is set up, you are ready to code once again! In the previous chapter, you built a static website with HTML and CSS, mainly focussing on structure and layout. In this chapter, you will learn to use SCSS in combination with BEM to give your stylesheets an underlying naming system, which will give your code even more structure and prepare you to build components in biotope later on. 
+
+### Challenge
+
+Build a so called 'call to action', consisting of a image, text and button. The component has 2 modifiers: red headline and secondary conversion, which has a completely different layout and look (see the screen below for reference). 
+
+<img style={{width: '100%'}} src={screen}/>  
+
+
+### Hints and guidelines
+
++ Adhere to the BEM naming structure closely as it is absolutely vital to learn it in order to write good biotope components later on
++ Use the BEM way to make modifiers for the component (see reference)
+
+### Resources
++ [SASS's own reference](https://sass-lang.com/guide) or [CSS-Tricks](https://css-tricks.com/snippets/sass/)
++ [BEM's own reference](http://getbem.com/naming/) or [CSS-Tricks](https://css-tricks.com/bem-101/)
+
+## JavaScript
+
+'Finally', is what you're thinking to yourself probably. Yes, now we're getting into the good stuff. Up until now, everything you have coded is more or less 'static'. Well, that changes now. In this section, you will get to Javascript, it's newest version ECMAScript2015 (also called ES6) and, to a lesser extent, jQuery. We here at VI try to write our code in ES6 whenever possible. jQuery, while it used to be the standard and provides a lot of functionality that makes our lives easier, is being used less and less and it's more than likely that it will completely die eventually. That is why coding in 'plain' JS is always preferred.
```

Comment:
We try to write Typescript if possible, so maybe we should add that here somewhere i think 👍

Reply:
Typescript is part of a later chapter. I am leaving it out here.